cmd/snap-confine/user-support.h

```diff
@@ -19,7 +19,5 @@
 #define SNAP_CONFINE_USER_SUPPORT_H
 
 void setup_user_data(void);
-void setup_user_xdg_runtime_dir(void);
-void mkpath(const char *const path);
```

Comment:
Is this a mistake?

Reply:
nope, we don't have mkpath(), it was removed in 0c4071a9b8717cbbdf11870be6624b0ceeacd07a